tests/__init__.py

```diff
@@ -8,6 +9,7 @@ def setUp(self):
         self.__patches = []
 
         self.error = NotImplemented
+        settings.NORMALIZE = False
```

Comment:
Initially I thought settings normalize flag should be set to False. Reverted the change. Thanks.